.release-please-manifest.json

```diff
@@ -1,19 +1,19 @@
 {
-  ".": "2.6.0",
-  "packages/api-explorer": "0.9.35",
-  "packages/code-editor": "0.1.22",
-  "packages/extension-api-explorer": "22.10.0",
-  "packages/extension-sdk": "22.10.0",
-  "packages/extension-sdk-react": "22.10.0",
-  "packages/extension-utils": "0.1.12",
-  "packages/hackathon": "22.10.0",
-  "packages/run-it": "0.9.35",
-  "packages/sdk": "22.10.0",
-  "packages/sdk-codegen": "21.6.0",
-  "packages/sdk-codegen-scripts": "21.4.1",
+  ".": "2.7.0",
+  "packages/api-explorer": "0.9.36",
+  "packages/code-editor": "0.1.23",
+  "packages/extension-api-explorer": "22.10.1",
+  "packages/extension-sdk": "22.10.1",
+  "packages/extension-sdk-react": "22.10.1",
+  "packages/extension-utils": "0.1.13",
+  "packages/hackathon": "22.10.1",
+  "packages/run-it": "0.9.36",
+  "packages/sdk": "22.11.0",
+  "packages/sdk-codegen": "21.7.0",
+  "packages/sdk-codegen-scripts": "21.4.2",
   "packages/sdk-codegen-utils": "21.0.11",
-  "packages/sdk-node": "22.10.0",
+  "packages/sdk-node": "22.10.1",
   "packages/sdk-rtl": "21.3.4",
-  "packages/wholly-sheet": "0.5.31",
+  "packages/wholly-sheet": "0.5.32",
   "python": "22.10.0"
```

Comment:
This is wrong, too. Should be "22.10.1"

Reply:
you're saying python should bump? were there non "chore" changes under the `python/` dir?

Reply:
So will python attempt to deploy again? If not, then it doesn't matter.

Reply:
[22.10 already exists](https://pypi.org/project/looker-sdk/22.10.0/) so it will not deploy again. I guess my question is whether you think there were changes that *should* have bumped python to 22.10.1 and didn't.